test/integration/attachment.test.js

```diff
@@ -103,8 +103,8 @@ describeAuthOnly('attachments - authenticated', function () {
   })
 
   it('should return 406 when requesting json from the octet route', async function () {
-    const attachment = '00000000-0000-1000-8000-000000000001'
-    const response = await getAttachmentRouteJSON(attachment, app, authToken)
+    const attachment = '00000000-0000-1000-9000-000000000001'
+    const response = await getAttachmentRouteOctet(attachment, app, authToken)
```

Comment:
I don't think this is right? The test is checking what happens is we request JSON from the octet-stream value (which looking at the seed is `00000000-0000-1000-8000-000000000001`)

Reply:
Mmm... I think you are correct. @nitro-marky any ideas?

Reply:
right which is JSON. This route should request JSON but from an octet stream only source. That's `00000000-0000-1000-8000-000000000001`